docs/api/clipboard.md

```diff
@@ -49,14 +62,9 @@ Writes the `image` into clipboard.
 
 Clears everything in clipboard.
 
-## clipboard.has(format[, type])
```

Comment:
The format parameter in `has` api is a bit misleading as its actually deduced from the string passed, should we make it clear in the docs ?